nginx/nginx.conf.ctmpl

```diff
@@ -0,0 +1,54 @@
+user  nginx;
+worker_processes  1;
```

Comment:
Just for reference: https://github.com/autopilotpattern/nginx/issues/2 includes a config file I've used previously for a high-volume site.